src/magicbox/SuggestionsManager.ts

```diff
@@ -86,12 +89,24 @@ export class SuggestionsManager {
     $$(this.root).trigger(OmniboxEvents.querySuggestLoseFocus);
   }
 
-  public moveDown(): Suggestion {
-    return this.returnMoved(this.move('down'));
+  public moveDown() {
```

Comment:
I think this function is doing too much. My intuition would be for methods named `moveDown`, `moveUp` to be `void`. If I want to get the current element, we should expose a public getter.

Reply:
Great. I'll see what I can do